src/regfox/responses/regfox_api_get_registration_info_parser.js

```diff
@@ -0,0 +1,329 @@
+
+/**
+ * Brings some of the "dynamic" data objects and places them directly on the main response.
+ * 
+ * So instead of 
+ * {
+ *  id: 1234
+ *  data: [{
+ *    path: "dateOfBirth"
+ *    value: "2022-02-02"
+ *  }]
+ * }
+ * 
+ * We get
+ * {
+ *  id: 1234,
+ *  dataOfBirth: "2022-02-02"
+ * }
+ * 
+ * @param {*} rawResponse see below
+ * @returns 
+ */
+const parseRegfoxGetRegistrationResponse = (rawResponse) => {
+  const dataObject = rawResponse?.data?.reduce((acc, i) => {
+    if (!i.value) {
+      return acc;
+    }
+    return { [i.path]: i.value, ...acc };
+  }, {});
+  return { ...dataObject, ...rawResponse };
+}
+
+export { parseRegfoxGetRegistrationResponse };
+
+// The raw response looks something like the below:
+/*{
```

Comment:
Worth dropping into a .json file in the tests directory as "this is what a real API result looks like" ?

Reply:
Its not currently legal json. If there is a "foo" account I can just grab some fake data from, happy to do it yeah!